client/js/screens/EditProfile/styles.js

```diff
@@ -0,0 +1,42 @@
+import theme from "../../config/globalStyles";
+
+const styles = {
+  header: {
+    marginVertical: 40,
+    marginHorizontal: 30,
```

Comment:
That's too much margin around your header. Can you use flexbox properties?

Reply:
Done.